wdio.conf.js

```diff
@@ -0,0 +1,214 @@
+        tags: require('./test/tagProcessor'),           // <string[]> (expression) only execute the features or scenarios with tags matching the expression
+        timeout: 20000,     // <number> timeout for step definitions
+        ignoreUndefinedDefinitions: false, // <boolean> Enable this config to treat undefined definitions as warnings.
+    },
+
+    //
+    // =====
+    // Hooks
+    // =====
+    // WebdriverIO provides several hooks you can use to interfere with the test process in order to enhance
+    // it and to build services around it. You can either apply a single function or an array of
+    // methods to it. If one of them returns with a promise, WebdriverIO will wait until that promise got
+    // resolved to continue.
+    //
+    // Gets executed once before all workers get launched.
+    // onPrepare: function (config, capabilities) {
+    // },
+    //
+    // Gets executed before test execution begins. At this point you can access all global
+    // variables, such as `browser`. It is the perfect place to define custom commands.
+    before: function () {
+        /**
+         * Setup the Chai assertion framework
+         */
+        var chai = require('chai');
+
+        global.expect = chai.expect;
+        global.assert = chai.assert;
+        global.should = chai.should();
+    },
+    //
+    // Hook that gets executed before the suite starts
+    // beforeSuite: function (suite) {
+    // },
+    //
+    // Hook that gets executed _before_ a hook within the suite starts (e.g. runs before calling
+    // beforeEach in Mocha)
+    // beforeHook: function () {
+    // },
+    //
+    // Hook that gets executed _after_ a hook within the suite starts (e.g. runs after calling
+    // afterEach in Mocha)
+    // afterHook: function () {
+    // },
+    //
+    // Function to be executed before a test (in Mocha/Jasmine) or a step (in Cucumber) starts.
+    // beforeTest: function (test) {
+    // },
+    //
+    // Runs before a WebdriverIO command gets executed.
+    // beforeCommand: function (commandName, args) {
+    // },
+    //
+    // Runs after a WebdriverIO command gets executed
+    // afterCommand: function (commandName, args, result, error) {
+    // },
+    //
+    // Function to be executed after a test (in Mocha/Jasmine) or a step (in Cucumber) starts.
+    // afterTest: function (test) {
+    // },
+    //
+    // Hook that gets executed after the suite has ended
+    // afterSuite: function (suite) {
+    // },
+    //
+    // Gets executed after all tests are done. You still have access to all global variables from
+    // the test.
+    // after: function (result, capabilities, specs) {
+    // },
+    //
+    // Gets executed after all workers got shut down and the process is about to exit. It is not
+    // possible to defer the end of the process using a promise.
+    // onComplete: function(exitCode) {
+    // }
+};
+
+require("babel-register");
```

Comment:
I think it would be cleaner to put this into the compiler array in the cucumberOpts